charts/ceph-csi-cephfs/Chart.yaml

```diff
@@ -1,10 +1,10 @@
 ---
 apiVersion: v1
-appVersion: canary
+appVersion: v2.1.0
 description: "Container Storage Interface (CSI) driver,
 provisioner, and attacher for Ceph cephfs"
 name: ceph-csi-cephfs
-version: 1.3.0-canary
+version: 2.1.0-canary
```

Comment:
why the mix of version v2.1.0 and 2.1.0-canary? Can it not all be the same version?

I would also expect v2.1-canary, not v2.1.0-canary...

Reply:
@nixpanic deploy.sh will replace will updates canary when pushing charts https://github.com/ceph/ceph-csi/blob/master/deploy.sh#L10

Reply:
In that case, should -canary not be used everywhere? Seems like version v2.1-canary can be used then.